test/src/code_field/code_controller_readonly_test.dart

```diff
@@ -197,7 +197,10 @@ void main() {
           const TextEditingValue(
             text: 'abc\nro\nro\nabc\n',
             //        \ cursor
-            selection: TextSelection.collapsed(offset: 3),
+            selection: TextSelection.collapsed(
+              offset: 3,
+              affinity: TextAffinity.upstream,
+            ),
```

Comment:
This test is failing because of the new version :/
For now I changed affinity to be as the actual result.

Still trying to find the reason.